webapps/frontend/app/viewmodels/site/vpool/vpool-detail.js

```diff
@@ -252,7 +242,7 @@ define([
             if (self.srSDMap().hasOwnProperty(sr.guid())) {
                 var wizard = new ReconfigureVPool({
                         modal: true,
-                        completed: deferred,
+                        completed: false,
```

Comment:
Not necessary anymore. Completed is part of the base wizard, no longer explicitly provided.